indico/htdocs/sass/partials/_toolbars.scss

```diff
@@ -156,7 +157,7 @@ $toolbar-thin-height: 1.9em;
     @extend %toolbar-group-slider;
     display: inline-block;
     padding: 0 0.4em;
-    margin-bottom: -2px;
+    margin-bottom: 0.2em;
```

Comment:
I agree that negative margins tend to be messy, but many times is the only way to make things look the way you want without refactoring many components depending on the padding of the container. According to my own [commit](https://github.com/indico/indico/commit/19cb82be6f2225d3022170b80279d9d814483b1e) in 2013, that negative margin fixed some glitches, but I can't recall what they were exactly. Maybe you can check on places were we use `.toolbar.group` and see if anything breaks.

I know we've been using `em` for layout stuff, but that should change. I'd say we want `rem` rather than `px` and never `em` for layout stuff.

A final comment, I don't think we want toolbars wrapping unless you are on a tiny screen.